test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1631,26 +1759,89 @@ def change_zone_epd(model, params)
           elec = OpenStudio::Model::ElectricEquipment.new(elecdef)
           elec.setSpace(space)
           elec.setName(elec_name)
           elec.setMultiplier(1)
           elec.setSchedule(elec_sch)
-          eqp_after = elec_eqp[0].getDesignLevel(space.floorArea,0)
+          eqp_after = elec_eqp[0].getDesignLevel(space.floorArea, 0)
           istop = 1
         end
       end
     end
     return model
+  end
 
+  # Remove cooling coil from air loops in model
+  # @param model [OpenStudio::model::Model] OpenStudio model object
+  # @param params [Array] zone_name, new equipment power density
+  # @return [OpenStudio::model::Model]
+  def remove_cooling_coils(model, params)
+    model.getAirLoopHVACs.each do |air_loop|
+      air_loop.supplyComponents.each do |supply_comp|
+        if supply_comp.iddObjectType.valueName.to_s.include?('OS_Coil_Cooling')
+          supply_comp.remove
+        elsif supply_comp.iddObjectType.valueName.to_s.include?('OS_AirLoopHVAC_UnitarySystem')
+          unitary_sys = supply_comp.to_AirLoopHVACUnitarySystem.get
+          cooling_coil = unitary_sys.coolingCoil
+          if cooling_coil.is_initialized
+            cooling_coil = cooling_coil.get
+            unitary_sys.resetCoolingCoil
+            cooling_coil.remove
+            controller_oa = air_loop.airLoopHVACOutdoorAirSystem.get.getControllerOutdoorAir
+            controller_oa.setEconomizerControlType('FixedDryBulb')
+          end
+        end
+      end
+    end
+    return model
+  end
+
+  # Add return and relief fans to air loops
+  # @param model [OpenStudio::model::Model] OpenStudio model object
+  # @param params [Array] zone_name, new equipment power density
+  # @return [OpenStudio::model::Model]
+  def return_relief_fan(model, params)
+    std = Standard.build('90.1-PRM-2019')
+    model.getAirLoopHVACs.each do |air_loop|
+      # Add return fan
+      return_fan = OpenStudio::Model::FanVariableVolume.new(model, model.alwaysOnDiscreteSchedule)
+      return_fan.setName("#{air_loop.name} return fan")
+      return_fan.addToNode(air_loop.returnAirNode.get)
+
+      # Add relief fan
+      relief_fan = OpenStudio::Model::FanVariableVolume.new(model, model.alwaysOnDiscreteSchedule)
+      relief_fan.setName("#{air_loop.name} relief fan")
+      relief_fan.addToNode(air_loop.reliefAirNode.get)
+
+      # Get supply fan
+      supply_fan = air_loop.supplyFan.get
+      if supply_fan.to_FanConstantVolume.is_initialized
+        supply_fan = supply_fan.to_FanConstantVolume.get
+      elsif supply_fan.to_FanVariableVolume.is_initialized
+        supply_fan = supply_fan.to_FanVariableVolume.get
+      elsif supply_fan.to_FanOnOff.is_initialized
+        supply_fan = supply_fan.to_FanOnOff.get
+      end
+
+      # Adjust return and relief fan power
+      # Get the current pressure rise (Pa)
+      return_fan.setPressureRise(supply_fan.pressureRise * 2)
+      relief_fan.setPressureRise(supply_fan.pressureRise * 3)
```

Comment:
Arbitrary value, definitely not realistic. Not sure if we care about it as long as the test is okay.